docs/index.md

```diff
@@ -1,6 +1,6 @@
 # API reference
 
-This is the documentation for the API v1 provided by the Cloudnet data portal.
+This is the documentation for the API v2 provided by the Cloudnet data portal.
```

Comment:
`/api/sites?modelSites` parametri pitäis kuvata ja `isModelOnlySite` lisätä esimerkki jsoniin.